src/modules/hip/kernel/glitch.hpp

```diff
@@ -0,0 +1,278 @@
+#include <hip/hip_runtime.h>
+#include "rpp_hip_common.hpp"
+
+template <typename T>
+__device__ __forceinline__ void rpp_hip_load1_glitch(T *srcPtr, uint2 srcStrideCH, float &locSrcX, float &locSrcY, float *dst, int channels)
+{
+    int srcIdx = locSrcY * srcStrideCH.y + locSrcX * srcStrideCH.x + channels;
+    rpp_hip_interpolate1_nearest_neighbor_load_pln1(srcPtr + srcIdx, dst);
+}
+
+template <typename T>
+__device__ __forceinline__ void rpp_hip_load8_glitch(T *srcPtr, uint2 srcStrideCH, d_float8 *srcX_f8, d_float8 *srcY_f8, d_float8 *dst_f8, int channels)
```

Comment:
@AryanSalmanpour: Can you please review this kernel?

Reply:
@rrawther Did the review on the call for this last week with @AryanSalmanpour
Added some comments in HIP kernels as per suggestions - https://github.com/ROCm/rpp/pull/357/commits/7d5855b7e36fb7e91c3ccc81fbd98d2c415c915d